models/qwen3/14b/decode_layer.py

```
@@ -1533,12 +1721,14 @@ def stack0(t, reps):  # replicate along dim 0
             print(f"[stacked-fwd {N}L+LMhead] swimlane perf run complete "
                   f"(host-ref argmax check skipped under --enable-l2-swimlane)")
             raise SystemExit(0)
-        # host ref: chain N -> final RMSNorm -> lm_head
+        # host ref: chain N single-layer decode_fwd_layers (N==1) -> final RMSNorm -> lm_head.
+        # Each step reloads the un-stacked single-layer inputs and overrides the hidden.
+        _CHUNK_NLAYERS = 1
         ref_hidden = inputs[0]; ref_out = None
```

Ruff (0.15.17)

[error] Multiple statements on one line (semicolon)

(E702)